### PR TITLE
Crosswords: Style updates

### PR DESCRIPTION
**Pull request by gtrufitt**
Style updates to improve contrast and readability of crosswords.
## Before

contrast-updates-before
## After

contrast-updates-after

Please shout if there's anything suspect regarding css architecture/variable use etc.

Ping/ @OliverJAsh @desbo 

This will close issue #10347

### CODE REVIEW

File: grunt-configs/px_to_rem.js [hunk 1]
```diff
@@ -4,7 +4,8 @@ module.exports = function(grunt, options) {
             options: {
                 map: options.isDev,
                 base: 16,
-                fallback: false
+                fallback: false,
+                ignore: ['stroke-width']
```
Comment (gtrufitt) [inline grunt-configs/px_to_rem.js]: Chrome doesn't recognise rems on SVGs - https://code.google.com/p/chromium/issues/detail?id=470449

**Comment by desbo** [comment]
:+1:

File: grunt-configs/px_to_rem.js (hunk 1 above)
Comment (OliverJAsh) [inline grunt-configs/px_to_rem.js]: Maybe leave this comment in the JS file

**Comment by OliverJAsh** [comment]
Nice work! :+1:

File: grunt-configs/px_to_rem.js (hunk 1 above)
Comment (gtrufitt) [inline grunt-configs/px_to_rem.js]: > Maybe leave this comment in the JS file

:+1: Totally. Done.

**Comment by gtrufitt** [comment]
Cheers. Will push this in first thing tomorrow.

**Comment by ScottPainterGNM** [comment]
This is brill!

200_s

sr4-gathell7

**Pull request merged**